docs/resources/cluster_rosa_hcp.md

```diff
@@ -75,6 +75,8 @@ resource "rhcs_cluster_rosa_hcp" "rosa_sts_cluster" {
 - `host_prefix` (Number) Length of the prefix of the subnet assigned to each node. After the creation of the resource, it is not possible to update the attribute value.
 - `kms_key_arn` (String) Used to encrypt root volume of compute node pools. The key ARN is the Amazon Resource Name (ARN) of a AWS Key Management Service (KMS) Key. It is a unique, fully qualified identifier for the AWS KMS Key. A key ARN includes the AWS account, Region, and the key ID(optional). After the creation of the resource, it is not possible to update the attribute value.
 - `machine_cidr` (String) Block of IP addresses for nodes. After the creation of the resource, it is not possible to update the attribute value.
+- `max_hcp_cluster_wait_timeout_in_minutes` (Number) This value sets the maximum duration in minutes to wait for a HCP cluster to be in a ready state.
```

Comment:
Might be a personal preference, but I think `max_wait_timeout_in_minutes` should be for the cluster control plane so that it is closer to classic. Than the second value we add max_machine_pool_wait...

Reply:
but then wouldn't ` max_machine_pool_wait` value be used for classic cluster creation, unless we duplicate ` max_machine_pool_wait` (60) and rename it for classic cluster creation it wouldn't make sense to have it named ` max_machine_pool_wait` and then use it for the wait time of a cluster.

Reply:
It would be the same attribute for waiting the cluster for both :thinking: , waiting the cluster on classic means it also waits for pools